zebra-rpc/src/methods.rs

```diff
@@ -1541,6 +1551,15 @@ impl Default for GetBlock {
 #[serde(transparent)]
 pub struct GetBlockHash(#[serde(with = "hex")] pub block::Hash);
 
+/// Response to a `getbestblockheightandhash` RPC request.
+#[derive(Copy, Clone, Debug, Eq, PartialEq, serde::Deserialize, serde::Serialize)]
+pub struct GetBlockHeightAndHash {
+    /// The best chain tip block height
+    pub height: block::Height,
+    /// The best chain tip block hash
+    pub hash: block::Hash,
+}
+
```

Comment:
Related to https://github.com/ZcashFoundation/zebra/pull/8616 i think it will be good to add a default for this response.

soemthing like this sdhould do it: 

```suggestion

impl Default for GetBlockHeightAndHash {
    fn default() -> Self {
        height: block::Height::MIN,
        hash: block::Hash([0; 32]),
    }
}
```

Reply:
Added in https://github.com/ZcashFoundation/zebra/pull/8596/commits/15feed173daced582340fdcdb4d4bb8cc9593a39.